src/tree/transform.js

```diff
@@ -64,9 +72,10 @@ export function setZooming(objectData, settingZoom, allowInteractions) {
         [0, 0],
         [width, height],
       ])
-      .filter(allowInteractions)
+      .filter(() => allowInteractions && event.type !== 'dblclick' && !(event.type === 'mousedown' && event.which === 3))
```

Comment:
This will have merge conflicts with the other PR :(
I guess I can merge master again